recipes/WSJ0Mix/meta/rir_constants.py

```diff
@@ -0,0 +1,8 @@
+NUM_BANDS = 4
```

Comment:
Please, add docstring with a description of the parameters + which other file is using it